build/NuSpecs/build-nupkg.ps1

```diff
@@ -9,7 +9,8 @@ Param(
     [string]$BuildFlavor = "release",
     [string]$BuildArch = "x86",
     [switch]$NoDeleteTemp,
-    [switch]$SkipFrameworkPackage
+    [switch]$SkipFrameworkPackage,
+    [switch]$SkipMakeNugetPackageAppxPackages
```

Comment:
Do we need this parameter? It's a little confusing to have both SkipFrameworkPackage and SkipMakeNugetPackageAppxPackages.

Maybe we could do something like: if SkipFrameworkPackage is false and the appx files don't exist, then make them. That way you avoid having to add a new param.

Reply:
That seems like a more reasonable way to handle this. Now the script checks for one specific file in the folder of the appx file for the given build arch and flavor and decides based on that.